config/main.py

```diff
@@ -7750,5 +7750,82 @@ def notice(db, category_list, max_events, namespace):
     handle_asic_sdk_health_suppress(db, 'notice', category_list, max_events, namespace)
 
 
+#
+# 'logrotate' group ('config logrotate ...')
+#
+@config.group(invoke_without_command=True)
+@click.pass_context
+@click.argument('file', required=True, type=click.Choice(['syslog', 'debug']))
+def logrotate(ctx, file):
+    """Configuring logrotate"""
+    # If invoking subcomand, no need to do anything
+    if ctx.invoked_subcommand is not None:
+        return
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.mod_entry(swsscommon.CFG_LOGGING_TABLE_NAME, file, None)
+
+
+@logrotate.command()
+@click.pass_context
+@click.argument('disk_percentage', metavar='<disk-percentage>',
+                required=True,  type=float)
+def disk_percentage(ctx, disk_percentage):
+    """Configuring logrotate disk-precentage"""
+    file = ctx.parent.params.get('file')
+    if 0 > disk_percentage > 100:
+        click.echo(f'Disk percentage {disk_percentage} is not in range [0 - 100]')
+        pass
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.mod_entry(swsscommon.CFG_LOGGING_TABLE_NAME, file,
+                        {'disk_percentage': disk_percentage})
+
+
+@logrotate.command(name='frequency')
+@click.pass_context
+@click.argument('frequency', metavar='<daily|weekly|monthly|yearly>',
+                required=True,
+                type=click.Choice(['daily', 'weekly', 'monthly', 'yearly']))
```

Comment:
As mentioned in sonic-buildimage PR we need frequency in minutes, hours as well

Reply:
Answered in `sonic-buildimage` repository